### ISSUE TITLE
hs19 photos

### ISSUE THREAD

**Issue by alexsweeten** [hackseq/hackseq19#31]
Photos from my camera are up on our Google Drive :) 

![IMG_1100](https://user-images.githubusercontent.com/7028404/67628802-9a24e400-f828-11e9-93e8-b0afd92acf80.JPG)

**Comment by baraaorabi** [comment]
Thanks for the awesome pics Alex!

@hackseq/hs19_organizers should we send a link to the pics to the participants in a post-hackseq19 wrap up email?

**Comment by ababaian** [comment]
That photo is like an album cover for a very nerdy punk band ;)

**Comment by jenjaelin** [comment]
thanks for sorting all the pics Alex, I definitely was just taking them haphazardly XD

**Comment by alexsweeten** [comment]
Your photos were great @jenjaelin. Unfortunately whoever took the group photo of everyone did a blurry job :(

**Comment by jenjaelin** [comment]
ah well. it happens. just keep the photo small for the site XD

**Comment by jenjaelin** [comment]
I need to photoshop Kristen and  Baraa (?) into each other's places to appease my OCD XD

**Comment by ababaian** [comment]
@SweetiePi Are you planning on updating the website with the final team photo?

**Comment by alexsweeten** [comment]
@ababaian Yes, I thought I did that already :face_with_head_bandage:. Will be up today.
